src/lib/ui/SupportedDevices/SupportedDevicesHeader.tsx

```diff
@@ -0,0 +1,14 @@
+export default function SupportedDevicesHeader() {
+  return (
+    <thead>
+      <tr>
+        <th></th>
+        <th>Category</th>
```

Comment:
@seveibar out of curiosity, does Guesty only want locks or will they later include some other devices? Should I add some `props` to allow these columns to be customized?

Reply:
We'll add other devices yea, I think "Category" will allow you to switch from e.g. smart locks to thermostats

Reply:
We can probably avoid customization for now i think- but maybe i'll eat my shoe